frontend/helpers.js

```diff
@@ -0,0 +1,20 @@
+export function getCurrentStage(gameState) {
```

Comment:
I feel like this helper method is going to be genius....I just haven't understood it yet